app/services/moderator/delete_user.rb

```diff
@@ -2,20 +2,20 @@ module Moderator
   class DeleteUser < ManageActivityAndRoles
     attr_reader :user, :admin, :user_params
 
-    def initialize(admin:, user:, user_params:)
-      @user = user
-      @admin = admin
-      @user_params = user_params
-    end
-
-    def self.call_deletion(admin:, user:, user_params:)
+    def self.call(admin:, user:, user_params:)
```

Comment:
Just renamed the method for the sake of consistency and moved the class method to the top.